app/models/card.rb

```diff
@@ -6,6 +6,20 @@ class Card < ApplicationRecord
 
   REVIEW_TIME_GAP = 3
 
+  def self.random
+    return nil if due_today.empty?
```

Comment:
@davetoxa ok. to clarify, what do you mean exactly by "left this line"? is this ok or do you want changes?